trestle/core/commands/remove.py

```diff
@@ -15,10 +15,113 @@
 # limitations under the License.
 """Trestle Remove Command."""
 
+import pathlib
+import warnings
+
 from ilcli import Command  # type: ignore
 
+import trestle.core.const as const
+import trestle.core.err as err
+from trestle.core import utils
+from trestle.core.models.actions import CreatePathAction, RemoveAction, WriteFileAction
+from trestle.core.models.elements import Element, ElementPath
+from trestle.core.models.file_content_type import FileContentType
+from trestle.core.models.plans import Plan
+from trestle.utils import fs
+
 
 class RemoveCmd(Command):
     """Remove a subcomponent to an existing model."""
 
     name = 'remove'
+
+    def _init_arguments(self):
+        self.add_argument(
+            f'-{const.ARG_FILE_SHORT}',
+            f'--{const.ARG_FILE}',
+            help=const.ARG_DESC_FILE + ' to remove component/subcomponent to.',
+            required=True
+        )
+        self.add_argument(
+            f'-{const.ARG_ELEMENT_SHORT}',
+            f'--{const.ARG_ELEMENT}',
+            help=const.ARG_DESC_ELEMENT + ' to remove.',
+            required=True
+        )
+
+    def _run(self, args):
+        """Remove an OSCAL component/subcomponent to the specified component.
+
+        This method takes input a filename and a list of comma-seperated element path. Element paths are field aliases.
+        The method first finds the parent model from the file and loads the file into the model.
+        Then the method executes 'remove' for each of the element paths specified.
+        """
+        args = args.__dict__
```

Comment:
You don't need to use args.__dict__. You can just use args.args_name.

Reply:
I'd have to use _argparse_, which is not used in **trestle add** either. Perhaps we can consider bringing in _argparse_ in a separate issue.

Reply:
The add arguments are argparse arguments. Check out trestle create for an example.